orcestra_book/ec_track.md

```diff
@@ -0,0 +1,71 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# EarthCARE Validation during ORCESTRA
+
+EarthCARE regularly flew over the measurement area, allowing each of the aircraft to fly underneath. The data from the underflights are used to validate the EarthCARE measurements. This page shows how to compare EarthCARE and HALO tracks.
+
+First, we load the track data. EarthCARE tracks are available for different regions of interest (`roi`) and forecast days.
+You can check [sattracks.orcestra-campaign.org](http://sattracks.orcestra-campaign.org) for an overview of all available forecasts.
+
+```{code-cell} ipython3
+from datetime import date
+
+import cartopy.crs as ccrs
+import cartopy.feature as cf
+import matplotlib.pyplot as plt
+import xarray as xr
+from orcestra import bco, sat
+
+
+# Get EartchCare track
+date = date(2024, 8, 25)
+ec_track = sat.SattrackLoader(
+    "EARTHCARE",
+    forecast_day=date,
+    kind="PRE",
+    roi="CAPE_VERDE",
+).get_track_for_day(date)
+```
+
+EarthCare tracks include the ascending and descending orbits.
+For the plot, we select only the afternoon overpass that coincided with our flight.
+
+```{code-cell} ipython3
+ec_track = ec_track.sel(time=slice(f"{date} 12:00", f"{date} 23:59"))
```

Comment:
That's a neat hack. Normally I'd recommend to do the calculation on datetime data types, but I have to admit that string concatenation looks nice in this case.